bundle/edu.gemini.wdba.xmlrpc.server/src/main/java/edu/gemini/wdba/tcc/TccNames.java

```diff
@@ -23,6 +23,7 @@ private TccNames() {
     public static final String AZEL = "AzEl";
     public static final String BASE = "Base";
 //    public static final String BASE = "sciencetarget";
+    public static final String BAGSTARGET = "bagsTarget";
```

Comment:
You are absolutely right. Since just the names are being passed, there is absolutely zero reason to expose the BAGS target to the TCC. I will instead simply include it with the list of all targets by using `GuideProbeTargets.getTargets` instead of `GuideProbeTargets.getManualTargets` and processing the BAGS target separately.

In that case, might be good to still test with TCC, but the behaviour with regards to `TargetGroupConfig` should be unchanged, essentially.